cmd/create/accountroles/cmd.go

```diff
@@ -250,19 +244,7 @@ func run(cmd *cobra.Command, argv []string) {
 			ocm.Version:  defaultPolicyVersion,
 		})
 	case aws.ModeManual:
-		err = aws.GeneratePolicyFiles(r.Reporter, env, true, true, policies, credRequests)
```

Comment:
This function is still necessary to generate the account role policies (note the first `true` sets `generateAccountRolePolicies` in the called function)

Reply:
How about moving this logic to operatorroles?

Reply:
ho yeah, great catch . we need it for manual mode policy creation for account role policies

Reply:
@vkareh I added this method back with some modifications. Please let me know how it looks like.

Reply:
Can you make it take a `nil` value instead? Allocating memory for an unused map doesn't seem right.

Reply:
Done.